torch/csrc/jit/codegen/cuda/scheduler/pointwise.cpp

```diff
@@ -145,7 +143,7 @@ c10::optional<PointwiseParams> getPointwiseHeuristics(
       std::max((lastPow2((int64_t)n_tensors) >> 2), (int64_t)1));
 
   // Don't unroll at the cost of getting a full wave on the GPU
-  if (n_elems < device_multiprocessor_count * kThreadX &&
+  if (n_elems > device_multiprocessor_count * kThreadX &&
```

Comment:
I don't think this should be flipped. Think that may be why the first test is vectorizing. Need to check.